docs/source/usage/usage.rst

```diff
@@ -1,139 +1,183 @@
+descriptor file. Once this file is present,
+you can proceed with Templates creation and Images conversion.
+These may be done in parallel, if you
 have appropriate compute resources.
 
 For an example of creating an image descriptor, see the
 :doc:`image settings creation example </examples/set_image_parameters>`.
 
 For more details about the available options, see the
-:doc:`detailed description of image parameters </usage/imageSettings>`, which also links
-to the relevant API calls for further information.
+:doc:`detailed description of image parameters </usage/imageSettings>`,
+which also links
+to the relevant parts of the CryoLike API for further information.
 
 
 Creating templates
 ------------------------
 
-CryoLike currently supports creating Templates from ``.mrc`` files (``.mrc``, ``.mrcs``, ``.map``)
-and from ``pdb`` files. Additionally, Templates may be created from physical-density data in
-other formats, if the data has already been loaded into memory as a Numpy Array or PyTorch Tensor.
+CryoLike currently supports creating Templates from
+``.mrc`` files (``.mrc``, ``.mrcs``, ``.map``) and from ``pdb`` files.
+Additionally, Templates may be created from physical-density data in
+other formats, if the data has already been loaded into memory as a
+Numpy Array or PyTorch Tensor.
 
-Template conversion will place the generated files in a standard directory structure and will
-additionally generate a list of the created files.
+Template conversion places the generated files in a standard directory
+structure and will additionally generate a list of the created files.
 
 For a basic example of generating Templates from PDB files, see the
 :doc:`template generation example </examples/make_templates>`.
 
 For more details about the available options for Template creation, see the
-:doc:`detailed description of Templates creation </usage/templateCreation>`, which includes
-details about the available parameters, supported formats, and links to the relevant API calls.
+:doc:`detailed description of Templates creation </usage/templateCreation>`,
+which includes details about the available parameters, supported formats,
+and links to the relevant parts of the CryoLike API.
 
 
 Converting images
 ------------------------
 
-CryoLike currently supports reading CryoSparc and Starfile image sources and images
-stored in ``.mrc`` format. The conversion
-process also handles restacking Images into standard sizes.
+CryoLike currently supports reading CryoSparc and Starfile descriptor
+files, and images stored in ``.mrc`` format. The conversion
+process also handles "restacking," or converting Image inputs into
+files with an equal number of images each.
 
-Converted Images stacks will be stored in a standard directory structure. Each stack will have
-its own parameters file recording provenance and capture data, as well as description of grids.
+Converted Images stacks are stored in a standard directory
```

Comment:
No substantive changes from here to end of document, except I added actual cross-references for the other sections.